tpr/error.go

```diff
@@ -4,23 +4,30 @@ import (
 	"github.com/juju/errgo"
 )
 
+var alreadyExistsError = errgo.New("already exists")
+
+// IsAlreadyExists asserts alreadyExistsError.
+func IsAlreadyExists(err error) bool {
+	return errgo.Cause(err) == alreadyExistsError
+}
+
 var invalidConfigError = errgo.New("invalid config")
 
 // IsInvalidConfig asserts invalidConfigError.
 func IsInvalidConfig(err error) bool {
 	return errgo.Cause(err) == invalidConfigError
 }
 
-var unexpectedlyShortResourceNameError = errgo.New("unexpectedly short resource name")
-
-// IsUnexpectedlyShortResourceName asserts unexpectedlyShortResourceNameError.
-func IsUnexpectedlyShortResourceName(err error) bool {
-	return errgo.Cause(err) == unexpectedlyShortResourceNameError
-}
-
 var tprInitTimeoutError = errgo.New("initialization timeout")
 
 // IsTPRInitTimeout asserts tprInitTimeoutError.
 func IsTPRInitTimeout(err error) bool {
 	return errgo.Cause(err) == tprInitTimeoutError
 }
+
+var unexpectedlyShortResourceNameError = errgo.New("unexpectedly short resource name")
+
+// IsUnexpectedlyShortResourceName asserts unexpectedlyShortResourceNameError.
+func IsUnexpectedlyShortResourceName(err error) bool {
+	return errgo.Cause(err) == unexpectedlyShortResourceNameError
+}
```

Comment:
@asymmetric I think you had a good suggestion to change that name, but I forgot.